tv_apps/browser/manifest.webapp

```diff
@@ -1,61 +1,210 @@
 {
   "name": "Browser",
-  "description": "Gaia Web Browser",
+  "description": "Browser",
   "launch_path": "/index.html",
-  "type": "certified",
-  "role": "system",
-  "developer": {
-    "name": "The Gaia Team",
-    "url": "https://github.com/mozilla-b2g/gaia"
-  },
+  "type": "privileged",
   "permissions": {
-    "themeable":{},
     "browser":{},
     "systemXHR":{},
-    "settings":{ "access": "readwrite" },
-    "geolocation" : {},
-    "desktop-notification" : {},
-    "audio-capture": {},
-    "video-capture": {},
-    "device-storage:pictures" : { "access": "readwrite" },
-    "device-storage:videos" : { "access": "readwrite" },
-    "device-storage:music" : { "access": "readwrite" }
+    "storage": {},
+    "desktop-notification" : {}
   },
   "locales": {
     "ar": {
-      "name": "المتصفح",
-      "description": "Gaia متصفح الويب"
+      "name": "Browser",
```

Comment:
Not sure if we need so many locales here... Maybe we just need en-US and some of them.